### ISSUE TITLE
Writing file

### ISSUE THREAD

**Issue by MaxDeBy** [xupefei/APNG.NET#5]
Any way to write APNG with this library?

I'm trying to find a way to write an APNG while specifying X and Y offsets but every tool or library I find just assumes every frame has the same dimensions as the default image. This makes optimizing animation files very hard.

**Comment by xupefei** [comment]
Use the "optimize" feature provided by https://sites.google.com/site/cphktool/apng-anime-maker.

**Comment by MaxDeBy** [comment]
Does that have CLI support? I need to automize the optimization process.

**Comment by xupefei** [comment]
I don't think so. You could try contacting the author for the source code and implement a CLI by yourself.
